diff -rupN dftd3.3.1.0/Makefile dftd3.3.1.0-new/Makefile
--- Makefile_orig	2014-06-30 07:25:50.000000000 -0400
+++ Makefile	2015-06-23 01:28:42.133487539 -0400
@@ -20,7 +20,7 @@ ifeq ($(OSTYPE),LINUXL)
 #  FC = gfortran
 # CC = gcc
 # LINKER = lf95
-  LINKER = ifort
+  LINKER = ifort  -static-libstdc++ -static-libgcc -static-intel -static
 #  LINKER = gfortran
   PREFLAG = -E -P
   CCFLAGS = -O -DLINUX 
